Tests/ReactiveSwiftTests/PropertySpec.swift

```diff
@@ -285,7 +285,7 @@ class PropertySpec: QuickSpec {
 
 					target! <~ source
 
-					queue.async(group: group) {
+					queue.async(group: group, flags: .barrier) {
```

Comment:
I think it does:

> Instead, the queue waits until its currently executing blocks finish executing. At that point, the barrier block executes by itself. Any blocks submitted after the barrier block are not executed until the barrier block completes.

(https://developer.apple.com/documentation/dispatch/1452797-dispatch_barrier_async?language=objc)

I think you'd need a separate, empty barrier block.